bangazonapi/views/paymenttype.py

```diff
@@ -81,10 +81,10 @@ def list(self, request):
         """Handle GET requests to payment type resource"""
         payment_types = Payment.objects.all()
 
-        customer_id = self.request.query_params.get('customer', None)
+        customer = request.auth.user
```

Comment:
You'll want to get the customer object that's logged in using the `request.auth.user` take a look at this line to see how to do that https://github.com/nss-day-cohort-48/bangazon-kritikillz931/blob/8d872d22505300b54a831aa978613167ac7340c0/bangazonapi/views/paymenttype.py#L39

Reply:
oh man. *face palm* changes made and pushed